generalInfoPage/general.js

```diff
@@ -1,5 +1,69 @@
 var menu = document.getElementById("display-menu");
-
+let selectedCountry = localStorage.getItem("selectedCountry");
+const Infocontainer = document.querySelector(".info-div");
+const InfoSection = document.querySelector(".info-section");
 function show() {
-    menu.classList.toggle('hide')
-  }
\ No newline at end of file
+  menu.classList.toggle('hide')
+} 
+const generateImage = (country) => {
+  fetch(
+    `https://api.codetabs.com/v1/proxy/?quest=https://imsea.herokuapp.com/api/1?q=${country}`
```

Comment:
If the user was in the universities page and he ack to the general page, what do you expect to have?